packages/contractkit/src/identity/odis/query.ts

```diff
@@ -136,6 +137,10 @@ export async function queryOdis<ResponseType>(
         case 401:
           throw new Error(ErrorMessages.ODIS_AUTH_ERROR)
         default:
+          if (Math.floor(res.status / 400) === 1) {
```

Comment:
This would be true for anything > 400 right? Including 500s

Reply:
A common approach that I've seen is 
`if (res.stats >= 400 && res.status < 500)`

Reply:
😂 Yep! I'll fix it